test/e2e/scheduler_predicates.go

```diff
@@ -110,6 +110,22 @@ func verifyResult(c *client.Client, podName string, ns string) {
 			"reason":                   "FailedScheduling",
 		}.AsSelector())
 	expectNoError(err)
+	// If we failed to find event with a capitalized first letter of reason
+	// try looking for one starting with a small one for backward compatibility.
+	// If we don't do it we end up in #158061.
+	// TODO: remove this block when we don't care about supporting v1.0 too much.
```

Comment:
Please file a separate issue from #15806 that is "Drop support for lowercased reason objects" or something similar, label it with `kind/breaking-binary-change`, and link to that, rather than just the issue for the 1.1 release.  Sorry that wasn't clear.  We're trying to keep track of where we needed hacks to get 1.1 out the door.

Also please use the `TODO(12345)` syntax.